source/renderer/app/containers/settings/categories/SupportSettingsPage.js

```diff
@@ -39,7 +39,8 @@ export default class SupportSettingsPage extends Component<InjectedProps> {
   };
 
   handleDownloadLogs = () => {
-    const { app } = this.props.actions;
+    const { app, networkStatus } = this.props.actions;
+    networkStatus.logStateSnapshot.trigger();
```

Comment:
@DeeJayElly are we 100% what this action does will finish before the packing/download log starts?

Reply:
Done